### DISCUSSION TITLE
The error_log directive in /usr/local/etc/php/conf.d/error-logging.ini does not make any difference

### DISCUSSION THREAD

**Discussion by siddjain** [docker-library/wordpress#596] [General]
The `wordpress:php7.4-fpm-alpine` image comes with following file. [ref](https://github.com/docker-library/wordpress/blob/master/latest/php7.4/fpm-alpine/Dockerfile#L69):

```
bash-5.0# cat /usr/local/etc/php/conf.d/error-logging.ini
error_reporting = E_ERROR | E_WARNING | E_PARSE | E_CORE_ERROR | E_CORE_WARNING | E_COMPILE_ERROR | E_COMPILE_WARNING | E_RECOVERABLE_ERROR
display_errors = Off
display_startup_errors = Off
log_errors = On
error_log = /dev/stderr
log_errors_max_len = 1024
ignore_repeated_errors = On
ignore_repeated_source = Off
html_errors = Off
```

According to official [PHP documentation](https://www.php.net/manual/en/function.error-log.php):

> message is sent to PHP's system logger, using the Operating System's system logging mechanism or a file, depending on what the error_log configuration directive is set to. This is the default option.

except that we haven't seen this work in practice. We changed the `error_log` to `/var/www/html/wp-content/test.log` but log messages are still written to `/var/www/html/wp-content/debug.log` which is the default location of [WP_DEBUG_LOG](https://wordpress.org/support/article/debugging-in-wordpress/#wp_debug_log). we thought maybe this is happening because the file does not exist so we created the file and even `chown` the permissions to make `www-data` the owner but still no luck. Has anyone experienced this?

**Comment by tianon** [comment]
My guess is that https://github.com/WordPress/WordPress/blob/fc6e4cbe3ef1dcd3a9977272d7ddcc413acdde0d/wp-includes/load.php#L446-L469 is related -- have you set `WORDPRESS_DEBUG`?

**Comment by siddjain** [comment]
Yes that explains it. Thanks a lot @tianon !
